BUILD.bazel

```diff
@@ -6,8 +6,29 @@ load("@rules_pkg//:mappings.bzl", "pkg_attributes", "pkg_files", "strip_prefix")
 load("@rules_pkg//:pkg.bzl", "pkg_tar", "pkg_zip")
 load("//:protobuf_javascript_release.bzl", "package_naming")
 
+config_setting(
+    name = "x64_x86_windows",
+    values = {"cpu": "x64_x86_windows"},
+)
+
+config_setting(
+    name = "x64_windows",
+    values = {"cpu": "x64_windows"},
+)
+
+config_setting(
+    name = "k8",
```

Comment:
Technically not required for the windows build but it doesn't hurt anything so I left it here so you can strill build the linux-x86_64 build using `--cpu k8`